### PR TITLE
fixed: Missing folder structure in html output #50

### PR DESCRIPTION
**Pull request by devCrossNet**
I hope the changes will solve the issue: Missing folder structure in html output #50

### CODE REVIEW

**Comment by devCrossNet** [comment]
@kitsonk is it possible to merge this pull request in the next days? or is there a reason why it takes so long?

**Comment by bryanforbes** [comment]
I just tried the patch with a webpack bundle of several unit tests plus supporting code and it generates an invalid `fileName`:

```
origSourceFilename: webpack:///webpack/bootstrap de2b4ff8ec450d37b9d3?60be
origFileName: unit.bundle.js
fileName: /path/to/dist.test/unit.bundle
```

It then throws an error when `originalPositionFor()` is called at line 76 of `remap.js`.

**Comment by devCrossNet** [comment]
hey @bryanforbes, can you provide an example of your setup? does remap work as expected without my patch or do you also have the issue #50 ?

**Comment by delasteve** [comment]
Could this be related?

https://github.com/SitePen/remap-istanbul/issues/68

**Comment by devCrossNet** [comment]
@delasteve @bryanforbes the problem was a missing file extension. I fixed that bug but I did not fix #68 . But I have a working setup to fix #68 . Maybe I will make another PR if I have enough time this weekend.

**Comment by Blackbaud-PaulCrowder** [comment]
I'd like to see this merged in, too, since we're currently referencing @devCrossNet's fork.

File: lib/remap.js [hunk 1]
```diff
@@ -231,6 +231,17 @@ define([
 				/* if there are inline sources and a store to put them into, we will populate it */
 				var inlineSourceMap = {};
 				if (sourceMap.sourcesContent) {
+					var origSourceFilename = rawSourceMap.sources[0],
```
Comment (dinoboff) [inline lib/remap.js]: You should declare the variable the `if` block for consistency.

Comment (devCrossNet) [inline lib/remap.js]: I'll change that asap

Comment (devCrossNet) [inline lib/remap.js]: done!

**Comment by delasteve** [comment]
@bryanforbes Any chance you could take a look at this again?

**Comment by jdonaghue** [comment]
@devCrossNet Thanks for your work here, looks great!

**Comment by devCrossNet** [comment]
@jdonaghue can you tell me in which version this fix will be released? and when? regards

**Pull request merged**